kuma/search/utils.py

```diff
@@ -69,24 +65,3 @@ def clean_params(self, params):
             if param and default:
                 clean_params[param] = default
         return clean_params
-
-
-def search_exception_handler(exc, context):
-    # Call REST framework's default exception handler first,
-    # to get the standard error response.
-    response = exception_handler(exc, context)
-
-    if response is None:
-        if isinstance(exc, elasticsearch.ElasticsearchException):
-            # FIXME: This really should return a 503 error instead but Zeus
-            # doesn't let that through and displays a generic error page in that
-            # case which we don't want here
```

Comment:
Zeus was the load balancer Mozilla used in the era before AWS. Now we use CloudFront and it's much better. 
Perhaps someday we "resurrect" this code so that if you use the rest framework you get an JSON error if an server-error was to happen.